functions/transformations/limit.go

```diff
@@ -155,7 +155,7 @@ func (t *limitTransformation) Process(id execute.DatasetID, tbl flux.Table) erro
 		}
 		n -= count
 		lcr := sliceColReader{
-			ColReader: cr,
+			ColReader: arrow.ColReader(cr),
```

Comment:
This seems wrong to me 😅 

Is there any other method that allows to reslice an arrow array and append it as column?  
Should I stick with converting it manually to a slice?

Reply:
The arrow packages has ways to create slices from arrow arrays https://godoc.org/github.com/apache/arrow/go/arrow#example-package--Float64Slice

Reply:
I made a similar comment here: https://github.com/influxdata/flux/pull/566#pullrequestreview-186636287

Reply:
I should learn to browse godocs 😅 Thank you!